open-api/rest-catalog-open-api.yaml

```diff
@@ -1628,6 +1800,121 @@ components:
         - $ref: '#/components/schemas/OAuthClientCredentialsRequest'
         - $ref: '#/components/schemas/OAuthTokenExchangeRequest'
 
+    CounterResult:
+      type: object
+      required:
+        - unit
+        - value
+      properties:
+        unit:
+          type: string
+        value:
+          type: integer
+          format: int64
+
+    TimerResult:
+      type: object
+      required:
+        - time-unit
+        - count
+        - total-duration
+      properties:
+        time-unit:
+          type: string
+        count:
+          type: integer
+          format: int64
+        total-duration:
+          type: integer
+          format: int64
+
+    MetricResult:
+      anyOf:
+        - $ref: '#/components/schemas/CounterResult'
+        - $ref: '#/components/schemas/TimerResult'
+
+    Metrics:
+      type: object
+      additionalProperties:
+        $ref: '#/components/schemas/MetricResult'
+
+    ReportMetricsRequest:
+      type: object
+      required:
+        - report-type
+        - report
+      properties:
+        report-type:
+          type: string
+        report:
```

Comment:
I mentioned this before, but I think this should send the report information here, rather than using an additional layer. That's what we do for similar cases.

Reply:
should be fixed now